src/kvstore/NebulaStore.cpp

```diff
@@ -1234,6 +1234,27 @@ nebula::cpp2::ErrorCode NebulaStore::multiPutWithoutReplicator(GraphSpaceID spac
   return nebula::cpp2::ErrorCode::SUCCEEDED;
 }
 
+nebula::cpp2::ErrorCode NebulaStore::multiRemoveWithoutReplicator(
+    GraphSpaceID spaceId, const std::vector<std::string>& keys) {
+  auto spaceRet = space(spaceId);
+  if (!ok(spaceRet)) {
+    LOG(WARNING) << "Get Space " << spaceId << " Failed";
+    return error(spaceRet);
+  }
+  auto space = nebula::value(spaceRet);
+
+  DCHECK_EQ(space->engines_.size(), 1);
```

Comment:
Why is only one engines_ (only one dataPath) allowed here?

Reply:
because it's meta, and there is only one partition in it

Reply:
But indeed, the nebula store should not know who use it, I'll remove it.